bot/src/application/chart/builders/ChartHtmlBuilder.ts

```
@@ -0,0 +1,128 @@
+    legendSection += '</div>';
+    return legendSection;
+  }
+
+  /**
+   * Build error HTML
+   */
+  static buildErrorHtml(error: unknown, title?: string): string {
+    const errorTitle = title || 'Error rendering chart';
+    return `<html><body><h1>${errorTitle}</h1><p>${error}</p></body></html>`;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)